### PR TITLE
feat: control all comunity dependencies as poetry groups

### PR DESCRIPTION
**Pull request by svidiella**

### CODE REVIEW

File: libs/community/langchain_google_community/bigquery.py [hunk 1]
```diff
@@ -56,7 +56,8 @@ def load(self) -> List[Document]:
         except ImportError as ex:
             raise ImportError(
                 "Could not import google-cloud-bigquery python package. "
-                "Please install it with `pip install google-cloud-bigquery`."
+                "Please, install bigquery dependency group: "
+                "poetry install --with bigquery"
```
Comment (lkuligin) [inline libs/community/langchain_google_community/bigquery.py:60]: `pip install langchain-google-community[bigquery]` should be better, or am I missing anything?

Comment (svidiella) [reply libs/community/langchain_google_community/bigquery.py:60]: you are right, changing them

Comment (svidiella) [reply libs/community/langchain_google_community/bigquery.py]: done!

**Pull request merged**